.mypy.ini

```diff
@@ -32,6 +32,9 @@ ignore_missing_imports = True
 [mypy-tensorflow.*]
 ignore_missing_imports = True
```

Comment:
Since we install tf/torch now, their missing imports should be removed, right?

Reply:
It works on my local, but not on CI. I think this is a separate issue: https://github.com/microsoft/DeepGNN/runs/7963445411?check_suite_focus=true#:~:text=examples/pytorch/link_prediction/poolers.py%3A5%3A%20error%3A%20Cannot%20find%20implementation%20or%20library%20stub%20for%20module%20named%20%22torch%22

Reply:
Does your local python version match CI version?

Reply:
Yes and same package versions